swap_meet/clothing.py

```diff
@@ -1,2 +1,11 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Clothing(Item):
+
+    def __init__(self, id = None, fabric = "Unknown", condition = 0.0, age = 0.0):
```

Comment:
Spaces do not surround `=` when you're using default params.

```suggestion
    def __init__(self, id=None, fabric="Unknown", condition=0.0, age=0.0):
```